### ISSUE TITLE
Media Cards: Focus Indicator is hard to see

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#77]
The gold focus indicator is hard to see against the gold buttons, but is not a failure.
